[EDU-5093] update bucket journey (#1144)
aziontech/docs: src/content/docs/en/pages/store-journey/storage/update-bucket.mdx

---
title: Updating an Edge Storage bucket
description: Learn how to update the buckets in your account using the Azion API and Azion CLI.
permalink: /documentation/products/store/storage/update-buckets/
meta_tags: Edge Storage, Storage, Cloud, S3, Bucket, File Storage, Object Storage, API, Azion CLI, Update Buckets
namespace: docs_store_journey_storage_update_buckets
menu_namespace: storeMenu
---

import Tabs from '~/components/tabs/Tabs'

This guide walks you through updating an Edge Storage bucket using the Azion API and Azion CLI. It includes changing the permissions of a bucket, which can be crucial for managing access to your data. 

## Updating a bucket

<Tabs client:visible>
    <Fragment slot="tab.cli">Azion CLI</Fragment>
    <Fragment slot="tab.api">API</Fragment>

<Fragment slot="panel.cli">

## Requirements 

- [Azion CLI installed](/en/documentation/products/azion-cli/overview/#installing-azion-cli). 
- [A personal token configured](/en/documentation/devtools/cli/globals/#token).

> To create a bucket: 

Reviewer: ```suggestion
To update a bucket: 
```

---
title: Updating an Edge Storage bucket
description: Learn how to update the buckets in your account using the Azion API and Azion CLI.
permalink: /documentation/products/store/storage/update-buckets/
meta_tags: Edge Storage, Storage, Cloud, S3, Bucket, File Storage, Object Storage, API, Azion CLI, Update Buckets
namespace: docs_store_journey_storage_update_buckets
menu_namespace: storeMenu
---

import Tabs from '~/components/tabs/Tabs'

> In this guide, we will walk you through the process of updating an Edge Storage bucket using both the Azion API and Azion CLI. This includes changing the permissions of a bucket, which can be crucial for managing access to your data. 

Reviewer: ```suggestion
This guide walks you through updating an Edge Storage bucket using the Azion API and Azion CLI. It includes changing the permissions of a bucket, which can be crucial for managing access to your data. 
```

> 

Reviewer: ```suggestion
```

## Updating a bucket

<Tabs client:visible>
    <Fragment slot="tab.cli">Azion CLI</Fragment>
    <Fragment slot="tab.api">API</Fragment>

<Fragment slot="panel.cli">

## Requirements 

- [Azion CLI installed](/en/documentation/products/azion-cli/overview/#installing-azion-cli). 
- [A personal token configured](/en/documentation/devtools/cli/globals/#token).

> To create a bucket: 

Reviewer 2: update

```bash 
azion update edge-storage bucket
```

Azion will present a series of interactions, so the bucket can be updated. Run `azion update edge-storage bucket -h` for further information about the data that can be updated. 

</Fragment>

<Fragment slot="panel.api">

> In this example, the permission of the bucket is changed. 

Reviewer: ```suggestion
In this example, the permission of the bucket will be changed. 
```

Reviewer: This section explains how to do it, it isn't already changed
